BuildResidentialHPXML/measure.rb

```diff
@@ -1207,6 +1227,65 @@ def arguments(model)
     arg.setUnits('W/CFM')
     args << arg
 
+    arg = OpenStudio::Measure::OSArgument::makeBoolArgument('heat_pump_is_shared_system', true)
+    arg.setDisplayName('Heat Pump: Is Shared System')
+    arg.setDescription('Whether the heat pump is a shared system. If true, assumed to serve all the units in the building. This is valid only for a ground-to-air heat pump.')
+    arg.setDefaultValue(false)
+    args << arg
```

Comment:
I suggest removing this argument and changing heat_pump_type_choices to split the GSHP option into GSHP w/ individual loop vs GSHP w/ shared loop. Then you can automatically map IsSharedSystem in the HPXML file.